virtual_rainforest/core/base_model.py

```diff
@@ -262,6 +290,81 @@ def _check_required_init_vars(cls) -> None:
             LOGGER.error(to_raise)
             raise to_raise
 
+    @classmethod
+    def _check_time_bounds_units(cls) -> None:
+        """Check that the time bounds defined by each model have time units.
```

Comment:
This is repetitive - we could have:

```suggestion
    def _check_time_bounds_units(cls, which: str = "lower_bound_on_time_scale") -> Quantity
        """Check that the time bounds defined by each model have time units..."""
        # extract the property to test
        val = cls.get(which)
```